tests/test_dataset.py.orig

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 MONAI Consortium
```

Comment:
Seems this is an unnecessary file, could you please help remove it?

Reply:
Removed.

Perhaps .gitignore should be updated?